src/components/PlayField/PlayField.tsx

```diff
@@ -21,9 +21,8 @@ function PlayField() {
 
     if (openCardOne.imageUrl === openCardTwo.imageUrl) {
       setSwitchedCardIds(switchedCardIds.concat(openCardIds));
-    } else {
-      setOpenCardIds([]);
     }
+    setOpenCardIds([]);
```

Comment:
🎉

Reply:
In the big world, ideally, this change would have been a separate pr as it's not related to this feature. Just a note for future